setup.py

```diff
@@ -47,5 +47,5 @@
     url='https://github.com/infermedica/python-api',
     license='Apache 2.0',
     packages=find_packages(exclude=['examples']),
-    install_requires=['requests>=1.0.0'],
+    install_requires=['requests>=2.0.0'],
```

Comment:
in `requirements.txt` is still `requests>=1.0.0`

Reply:
Thanks for catching that, fixed :)